src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Dec2Hex.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // Dec2Hex(number:n, [places:n])
+    internal sealed class Dec2HexFunction : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        public override bool IsStateless => true;
+
+        public override bool SupportsParamCoercion => true;
+
+        public override bool HasPreciseErrors => true;
+
+        public Dec2HexFunction()
+            : base("Dec2Hex", TexlStrings.AboutDec2Hex, FunctionCategories.MathAndStat, DType.String, 0, 1, 2)
+        {
+        }
+
+        public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
+        {
+            yield return new[] { TexlStrings.Dec2HexArg1, TexlStrings.Dec2HexArg2 };
+        }
+
+        protected override bool CheckInvocation(CheckTypesContext context, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
```

Comment:
>CheckInvocation [](http://example.com/codeflow?start=32&length=15)

This function probably doesn't need to override CheckInvocation, since it doesn't do anything fancy with its type. If you remove this override, would anything break?